src/main/re_pressed/impl.cljs

```diff
@@ -146,78 +145,77 @@
    (register-events event-type)
    (register-effects event-type)
 
-   (.on (js/$ js/document)
-        event-type
-        (fn [e]
-          (let [ns-keyword (->ns-keyword event-type)
-
-                e-key         (.-which e)
-                modifier-key? (modifier-keys e-key)
-
-                ;; --
-                ;; thanks re-frame-10x for this snippet
-                tag-name        (.-tagName (.-target e))
-                entering-input? (contains?
-                                 #{"INPUT"
-                                   "SELECT"
-                                   "TEXTAREA"} tag-name)
-                ;; --
-
-                hit-key {:altKey   (.-altKey e)
-                         :ctrlKey  (.-ctrlKey e)
-                         :metaKey  (.-metaKey e)
-                         :shiftKey (.-shiftKey e)
-                         :which    (.-which e)}
```

Comment:
This diff is a bit big due to whitespace changes but essentially the change is that Google Closure doesn't have `which`, it normalizes `keyCode` instead.

Reply:
@heyarne We will need to rename `:which` to `:keyCode` here.